.github/workflows/codeql-analysis.yml

```
@@ -80,7 +80,8 @@ jobs:
           mvn -e -B -V org.apache.maven.plugins:maven-wrapper-plugin:3.1.0:wrapper "-Dmaven=3.8.6"
 
       - name: Clean install dependencies and build
-        run: mvn clean install -DskipTests -B -V
+        run: mvn clean install -DskipTests -B -V -Dcheckstyle.skip=true -Dlicense.skip=true -Dmaven.site.skip=true -Dmaven.javadoc.skip=true
```

How about this way instead ...

```suggestion
        run: mvn clean install -DskipTests -B -V -Pfast
```